### PR TITLE
feat(yard): hover tokens to show prop type

### PR DESCRIPTION
**Pull request by chasestarr**
#### Description

Screen Shot 2019-10-23 at 2 16 35 PM

Only added to `Accordion` for now. If we agree on this approach I can apply to the rest of components

### CODE REVIEW

**Comment by gergelyke** [comment]
this looks really great!

File: documentation-site/components/yard/yard.tsx [hunk 1]
```diff
@@ -277,6 +281,53 @@ const Yard: React.FC<
             updateCode(dispatch, newCode);
           }
         }}
+        transformToken={tokenProps => {
```
Comment (tajo) [inline documentation-site/components/yard/yard.tsx]: I would move it into a separate file to keep yard.tsx as short as possible

**Comment by tajo** [comment]
Works great! Should be this part of OSS version?

File: documentation-site/components/yard/type-definition.js [hunk 2]
```diff
@@ -0,0 +1,385 @@
+/*
```
Comment (tajo) [inline documentation-site/components/yard/type-definition.js]: Is this file from some other project? Should we have some tests?

Comment (chasestarr) [reply documentation-site/components/yard/type-definition.js]: yeah this is from the extract-react-types package. I've vendored the code for now, but going to put up a pr to that package to help out

**Comment by chasestarr** [comment]
hmm not sure if it should be in the initial OSS version we may want to see how stable/useful it is

File: documentation-site/components/yard/type-definition.js (hunk 2 above)
Comment (chasestarr) [reply documentation-site/components/yard/type-definition.js]: related pr: https://github.com/atlassian/extract-react-types/pull/101

**Comment by gergelyke** [comment]
👏

**Pull request merged**